src/actions/user_block_actions.js

```diff
@@ -0,0 +1,44 @@
+import axios from "axios";
+import {
+  FETCH_BLOCKED_USERS,
+  FETCH_BLOCKED_BY,
+  CREATE_BLOCK,
+  FETCH_USER_CONNECTIONS
+} from "./types";
+
+export const fetchBlockedUsers = userId => async dispatch => {
+  const request = await axios({
+    method: "get",
+    url: process.env.REACT_APP_API_ENDPOINT + "users/blocked",
+    headers: { id: userId }
+  });
+  dispatch({ type: FETCH_BLOCKED_USERS, payload: request });
+};
+
+export const fetchBlockedBy = userId => {
```

Comment:
This action is not being used by any component.

Reply:
@mariuscm you can remove this code since it's not being used anywhere in the app

Reply:
I removed the fetchBlockedBy action and pushed the changes